project/ws/app/src/lib/routes/user-profile/components/user-profile/user-profile.component.html

```diff
@@ -413,7 +414,13 @@
                         </mat-form-field>
                       </div>
 
-                      <div class="input-wrapper margin-top-s">
+                      <div class="input-wrapper" *ngIf="!isVerifiedAlready">
+                        <div class="flex flex-1 flex-middle">
+                          <img width="18px" height="18px" mat-raised-button src="assets/icons/info.svg" alt="">
+                          <span class="ml-2">Once you've filled in all the mandatory fields in your profile and receive approval, you will be recognized as a Verified Karmayogi.</span>
```

Comment:
Hardcoded vaues should come from constant

Reply:
resolved